src/test/java/com/stripe/functional/TelemetryTest.java

```diff
@@ -92,14 +106,16 @@ public void testTelemetryDisabled() throws StripeException, IOException, Interru
 
   @Test
   public void testTelemetryWorksWithConcurrentRequests() throws IOException, InterruptedException {
+    final JsonParser jsonParser = new JsonParser();
+
     @Cleanup MockWebServer server = new MockWebServer();
 
     for (int i = 0; i < 20; i++) {
       server.enqueue(
           new MockResponse()
               .setBody("{}")
               .addHeader("Request-Id", "req_" + i)
-              .setBodyDelay(50, TimeUnit.MILLISECONDS));
+              .setBodyDelay(100, TimeUnit.MILLISECONDS));
```

Comment:
The test was a little bit flaky. This makes it a bit slower but hopefully more consistent.